WSEvents.cpp

```diff
@@ -87,12 +87,15 @@ WSEvents::WSEvents(WSServer* srv) {
     QTimer::singleShot(1000, this, SLOT(deferredInitOperations()));
 
     Heartbeat_active = false;
-    Heartbeat_CurrentProfile_active=false;
-    Heartbeat_CurrentScene_active-false;
-    Heartbeat_Streaming_active=false;
+    Heartbeat_CurrentProfile_active = false;
+    Heartbeat_CurrentScene_active = false;
+    Heartbeat_Streaming_active = false;
     Heartbeat_Recording_active = false;
-    Heartbeat_TotalStreamTime_active = false;
-    Heartbeat_TotalRecordTime_active = false;
+	Heartbeat_TotalStreamTime_active = false;
```

Comment:
Incorrect indentation here (a tab instead of four spaces), if you'd like I can fix it for you.

Reply:
Done. There were also a few more in it.